storage/queue.go

```diff
@@ -263,10 +263,13 @@ func (bq *baseQueue) processOne(clock *hlc.Clock, stopper *util.Stopper) {
 			}
 		}
 		if err := bq.impl.process(now, rng); err != nil {
-			log.Errorf("failure processing range %s from %s queue: %s", rng, bq.name, err)
-		}
-		if log.V(1) {
-			log.Infof("processed range %s from %s queue in %s", rng, bq.name, time.Now().Sub(start))
+			if log.V(1) {
```

Comment:
We should be very cautious about silencing errors; this can very easily lead to problems that are nightmares to debug. Is the single-node "failure processing range range=1 ["" - "\xff\xff") from replicate queue: allocator.go:204: unable to allocate a target store; no candidates available" the only spammy case here? We should just make that case return nil instead of passing an error up the stack.